codechecker_lib/build_manager.py

```diff
@@ -37,11 +45,22 @@ def perform_build_command(logfile, command, context, silent=False):
         original_env = os.environ.copy()
 
     return_code = 0
-    # Run user's commands in shell
-    log_env = analyzer_env.get_log_env(logfile, context, original_env)
 
-    if 'CC_LOGGER_GCC_LIKE' not in log_env:
-      log_env['CC_LOGGER_GCC_LIKE'] = 'gcc:g++:clang:clang++:cc:c++'
+    # Run user's commands with intercept
+    if host_check.check_intercept(original_env):
+        LOG.debug_analyzer("with intercept ...")
+        final_command = command
+        command = "intercept-build " + final_command
+        log_env = original_env
+        LOG.debug_analyzer(command)
+
+    # Run user's commands in shell
+    else:
+        if platform.system() == 'Linux':
+            LOG.debug_analyzer("with ld logger ...")
+            log_env = analyzer_env.get_log_env(logfile, context, original_env)
+            if 'CC_LOGGER_GCC_LIKE' not in log_env:
+                log_env['CC_LOGGER_GCC_LIKE'] = 'gcc:g++:clang:clang++:cc:c++'
 
     LOG.debug_analyzer(log_env)
```

Comment:
Hi, I forgot about this option. I've made some changes, you can check it. Thanks for your help :) @r0mai